packages/suite/src/components/suite/layouts/SuiteLayout/PageHeader/HeaderActions.tsx

```diff
@@ -142,45 +104,16 @@ export const HeaderActions = () => {
             )}
 
             {isCoinmarketAvailable && (
-                <AppNavigationTooltip>
-                    <ShowOnLargeDesktopWrapper>
-                        <ButtonComponent
-                            icon="currencyCircleDollar"
-                            onClick={() => {
-                                goToWithAnalytics('wallet-coinmarket-buy', {
-                                    preserveParams: true,
-                                });
-                            }}
-                            data-testid="@wallet/menu/wallet-coinmarket-buy"
-                            variant="tertiary"
-                            size="small"
-                            isDisabled={isAccountLoading}
-                        >
-                            <Translation id="TR_COINMARKET_BUY_AND_SELL" />
-                        </ButtonComponent>
-                    </ShowOnLargeDesktopWrapper>
-                    {!hasBitcoinOnlyFirmware(device) && (
-                        <ButtonComponent
-                            icon="arrowsLeftRight"
-                            onClick={() => {
-                                goToWithAnalytics('wallet-coinmarket-exchange', {
-                                    preserveParams: true,
-                                });
-                            }}
-                            data-testid="@wallet/menu/wallet-coinmarket-exchange"
-                            variant="tertiary"
-                            size="small"
-                            isDisabled={isAccountLoading}
-                        >
-                            <Translation id="TR_COINMARKET_SWAP" />
-                        </ButtonComponent>
-                    )}
-                </AppNavigationTooltip>
+                <TradeActions
+                    analyticsEventType={EventType.AccountsActions}
+                    selectedAccount={selectedAccount}
+                    hideBuyAndSellBelowDesktop
```

Comment:
Why do do we hide it below desktop? Users will see just Swap on the dashboard on mobile
![Screenshot 2024-10-17 at 14 48 16](https://github.com/user-attachments/assets/350f6da9-967b-432f-baf8-dd753436d798)